lms/djangoapps/support/static/support/js/views/enrollment.js

```diff
@@ -39,12 +39,7 @@
                     user: user,
                     enrollments: this.enrollments,
                     formatDate: function(date) {
-                        if (!date) {
-                            return 'N/A';
-                        }
-                        else {
-                            return moment(date).format('MM/DD/YYYY (H:MM UTC)');
-                        }
+                        return date ? moment.utc(date).format('lll z') : 'N/A';
```

Comment:
@attiyaIshaque Why are we changing format from `10/01/2016 (5:10 UTC)` to `Jan 10, 2016 5:10 AM UTC`. I think `10/01/2016 5:10 AM UTC` would be better.